### ISSUE TITLE
Comment regarding "complete class" theorem attributed to Wald

### ISSUE THREAD

**Issue by droy** [probml/pml-book#210]
Wald never proved the theorem labelled 5.3.1: "Every admissible frequent decision procedure is bayes with respect to some possibly improper prior". As stated, I'm also fairly certain that this theorem has never been proven. It certaintly wasn't shown by Wald, because I believe he may have died before improper priors were really a thing, though limits of Bayes procedures 
 perhaps.  The first reference I know for generalized Bayes is [Sacks. "Generalized Bayes solutions in estimation problems." Ann. Math. Statist. 34 751–768, 1963], although Sacks suggests the idea has been around (in the air, I suppose) for a long while. He later refers to [Samuel Karlin. "Admissibility for Estimation with Quadratic Loss." Ann. Math. Statist. 29 (2) 406 - 436, June, 1958. https://doi.org/10.1214/aoms/1177706620], who points to work in 1950/1951 by Lehmann and by Bylth (Lehmann's student) on using limits of Bayes procedures. The latter describes an approach, now known as Blyth's method, for establishing admissibility via limits of Bayes procedures. This approach is not known to be complete, even if it is sound.

Note that a theorem that "Assuming XYZ, admissible => Bayes" is not technically a complete class theorem, because, in some problems, the class of all admissible procedures in not a complete class. Admittedly, these are strange decision problems usually, though not always. Wald did prove theorems of the form "admissible => Bayes" (as well as theorems that also established that the admissible procedures were complete) but they were for problems with finite parameter spaces or under conditions like compactness of the parameter space and continuity of the risk function. Of course, these were seminal theorems.

James Berger is really the one to be credited for proving "admissible => generalized Bayes" (and some complete class problems), but these results were only for exponential family models (+ some regularity). Indeed, in general, all existing theorems of the form "admissible => [some form of Bayes]" were, until recently, laden with regularity conditions of some sort.

Haosui Duanmu and I showed recently that "a procedure is extended admissible <==> its Bayes risk is within an infinitesimal of the Bayes optimal risk, with respect to a prior that may assign infinitesimal mass". This will appear in Annals of Stats.

       https://www.e-publications.org/ims/submission/AOS/user/submissionFile/31623?confirm=381d7eb7

Our characterization, in contrast to all previous work, has no regularity conditions and makes no strong assumptions on the problem (such as boundedness of the loss). So, in conclusion, Bayes'ness is slightly weaker than admissibility, since extended admissibility says that there is no procedure that has uniformly better risk (eps better everywhere, for some eps > 0).

In summary, the result credit'd to Wald is indeed very similar to results by Wald, but it is missing very strong assumptions that were only lifted 70 years later. Berger is due some credit here, but even his result has assumptions. If you want to handle any problem, you may need a prior assigning infinitesimal mass (our work).

**Comment by droy** [comment]
In some problems, you can show that the generalized Bayes rules are a
complete class. Exponential families on finite dimensional vector spaces
with nice losses are an example. I’d have to look at Berger’s paper to give
you a more precise statement. My feeling is that this is the sort of
theorem you would want to quote. Let me find one for you. And then you
could mention that, in general, the result does not hold but there is a
very general connection that holds without any conditions, but it involves
a notion of Bayes optimality allow infinitesimals probabilities.

Regarding the statement by Cox and Hinkley: let me dig this up and look at
the context. It is certainly false if you consider only proper Bayes rules
(i.e., those derived from proper priors). I suspect they are using Bayesian
to refer to a broader class. Indeed, since priors are really in the "mind"
of the statistician only, I felt that our solutios using infinitesimals was
rather palatable, since we're already used to improper priors and other
deviations. If I had to guess the class to which they are referring, it
might be generalized Bayes (so, rules derived from the "formal" (i.e.,
mechanical) application of Bayes rule to an improper density) and then they
would have just dropped a bunch of hypothesis. But Cox and Hinkley were
probably looking at exponential families and so that may have been the
context in which one must understand their statement.

What's the time frame you need me to sort out this statements in order to
meet your own deadlines?

Dan

**Comment by pglpm** [comment]
Besides these important results and theorems, I'd be glad if the book emphasized that "admissible" must not be taken in the common sense of the word, but as a technical term only and that an "admissible" decision rule can be silly, and an "inadmissible" one can be cogent and appropriate. Stressing this is important for the audience of the book, who's not limited to statisticians, and so may make dangerous psychological associations (as with the horrible term "significant" in statistics). The book gives a short warning about this on page 186, right after Theorem 5.3.1, but I wish the point were emphasized much more.

See also the interesting discussion about this in [Jaynes](https://doi.org/10.1017/CBO9780511790423), around § 13.7–13.10, especially pp. 409, 415; and the examples around § 17.5.

**Comment by droy** [comment]
I’ll push back: Our Annals paper shows that nonstandard Bayes if and only
if extended admissible. And so Jaynes, whether or not he likes to think
about admissibility, is always extended admissible because he is Bayes
(easy to prove direction) and every extended admissible is nonstandard
Bayes.

**Comment by pglpm** [comment]
Hi Daniel, I'm not questioning the validity of your proofs. The problem is with this kind of technical terms and particular readerships such as Kevin's. Does "extended admissible" include what was called "inadmissible" before? Does "extended admissible" now mean 'admissible' in the common sense of the term? And does "nonstandard Bayes" mean a Bayes that's more open to criticisms, or not agreed upon, or idiosyncratic? Imagine a clinician who's using machine-learning methods for cancer diagnosis and is studying from Kelvin's book to have a better understanding about them. Unfortunately terms like the above are not neutral, unlike "diffeomorphic" or "C*-algebra" or "sequent calculus".  Not rarely one has to end up explaining that "subjective" in "subjective Bayesian" doesn't mean "whimsical". So I'd be glad if Kevin warned his readers against attaching common-sense meanings to terms that only have a technical meaning; and, when the common-sense meaning actually applies, if he explained why.

**Comment by murphyk** [comment]
who knew that i would ignite a bayesian flame war inside of  github! 
Anyway, I fixed the typo "admissible" but otherwise have left my (new) text unchanged. It cites Dan and's paper and Jaynes's book, so readers can read both and then make up their own minds :)

**Comment by fortuin** [comment]
I just stumbled upon this discussion when googling things about admissibility and I found it quite entertaining and insightful to read, so I just wanted to say thanks for having it in the public domain :-)
